mirrord/layer/src/socket.rs

```diff
@@ -443,6 +443,9 @@ impl OutgoingSelector {
                     .filter(filter_protocol)
                     .any(any_address)
                 {
+                    // Our "local" selector matched, but this `address` might be a remotely resolved
```

Comment:
```suggestion
                    // Our "local" selector matched (e.g. because of the port), but this `address` might be a remotely resolved
```
Maybe add this to explain why it can be valid for a remotely resolved address to be matched by a local filter list?